### DISCUSSION TITLE
5,10, or 14 digit HIOS - Potential for all to be acceptable?

### DISCUSSION THREAD

**Discussion by brbucher** [CMSgov/price-transparency-guide#500] [Q&A] [answered]
Hello,
Moving this pull request ([#447](https://github.com/CMSgov/price-transparency-guide/pull/447)) discussion to an actual discussion thread since a conclusion wasn't reached.
Could 5, 10, and 14 digit HIOS be acceptable?
From the discussion in the pull request I understand the want for reducing duplication of data which many of the other revisions have done a good job of achieving, and many of those revisions have been optional. 
This revision could be more difficult for affected organizations to implement, as they may have to update all of their 14 digit HIOS ids to 10 digit HIOS.  After discussing with some organizations, the 10 digit HIOS is not a familiar concept to them.
It seems like it could be an either of the three option rather than a pick two.

**Comment by brbucher** [comment]
@shaselton any thoughts here?

**Comment by nmaylevc** [comment]
Now that there are table of contents files that allow the machine readable file with the in-network rates to be easily shared across multiple plans, it would seem that 14 digit HIOS IDs would create much less extra data.

**Comment by brbucher** [comment]
Is the expectation for the last 4 digits of the HIOS to be truncated to produce the 10 digit?

**Reply by bryanbriegel** [reply]
A 14 byte HIOS Plan ID describes the plan, where rolling back to first 10 bytes, those 10 bytes define the product, and it is the 10 bytes that are specified in the schema.

**Reply by shaselton-usds** [reply, marked as answer]
@brbucher -- yes. If using the 10-digit HIOS, it would be the first 10-digits of the 14 digit ID. Identifying the plan to the product level is 100% acceptable.

Here is what the 14-digit ID is comprised of:
* Issuer’s ID number,  (first 5 digits, e.g.  12345)
* the Issuer’s state abbreviation, (next 2 digits, e.g. WA)
* the Issuer’s three digit Product ID, and  (next 3 digits, e.g. 001)
* a four digit sequence number that is the “Component ID” (last 4 digits, e.g. 0001)
